fvm/src/syscalls/env.rs

```diff
@@ -0,0 +1,23 @@
+use super::Context;
+use crate::kernel::{Kernel, Result};
+
+pub fn timestamp(_context: Context<'_, impl Kernel>) -> Result<u64> {
+    todo!()
+}
+
+pub fn blockhash(
```

Comment:
This should be `tipset_cid`, the block hash is something different than what this should return.

Reply:
sure!